src/components/Calculator.js

```diff
@@ -1,125 +1,123 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import './Calculator.css';
 import calculate from '../logic/calculate';
 
-class Calculator extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      total: '',
-      next: '',
-      operation: '',
-    };
-    this.handleClick = this.handleClick.bind(this);
-  }
+const Calculator = () => {
```

Comment:
- [ ] There is a bug in your calculator that is preventing doing any operation.  I'll explain with a video:

https://share.getcloudapp.com/p9u65ggX

Please, kindly fix your calculator so all the operations are possible to be carried out again. 

I recommend looking at the console while trying to make the operations, so you can get hits of what is not working and why. 💪

Reply:
Fixed. Thank you